src/main.ts

```diff
@@ -154,7 +157,7 @@ export class Purchases {
    */
   static configure(
     apiKey: string,
-    appUserId: string,
+    appUserId: string | IdentityMode,
```

Comment:
The reason why we decided to specifically request to autogenerate anonymous user IDs instead of not passing an app user ID at all (like in the mobile SDKs) was so developers are more conscious about this choice.

Reply:
this makes me feel like it's the way it should always have been in mobile too 😅

Reply:
I like it and dislike it at the same time 😅 
The lazy me would like to just avoid to pass the parameter.
The functional programming me (which is usually less lazy) agrees with this implementation.

Proposing this change:
```suggestion
    appUserId: string | IdentityMode = IdentityMode.Anonymous,
```

In this way we make everyone happy:
```
// Lazy me
purchases.configure(API_KEY)

// Less lazy me
purchases.configure(API_KEY, IdentityMode.Anonymous)
```

Wdyt?

Reply:
The most annoying part I see with this is the usual TS/JS limitation when using named parameters.

We will always require users to set a value for `appUserId` just to be able to set the following parameter.

```
Purchases.configure(API_KEY, IdentityMode.Anonymous, { ...options})
```

But not so much we can do about it without breaking the previous usages.

Reply:
> In this way we make everyone happy:
>
> ```
>// Lazy me
>purchases.configure(API_KEY)
>
>// Less lazy me
>purchases.configure(API_KEY, IdentityMode.Anonymous)
>```
>Wdyt?

The thing is that we want developers to specifically opt in autogenerated anonymous IDs. In the web, is more risky since these anonymous IDs are not persisted in the client in any way AFAIK.